boards/xtensa/mt8195_adsp/mt8195_adsp.dts

```diff
@@ -0,0 +1,93 @@
+/* Copyright 2023 The ChromiumOS Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <mem.h>
+
+/dts-v1/;
+/ {
+
+#address-cells = <1>;
+#size-cells = <1>;
+
+sram0: memory@40000000 {
```

Comment:
I think all this stuff should be in `dts/xtensa/mediatek/mt8195.dtsi` or something like that with most peripherals disabled, then this file should include that, enabled whatever peripherals are on the specific board, set the uart, chosen, aliases etc..